libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,572 @@
+  }
+
+}
+
+package com.mapbox.navigation.dropin.component.tripsession {
+
+  public abstract sealed class TripSessionStarterAction {
+  }
+
+  public static final class TripSessionStarterAction.EnableReplayTripSession extends com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction {
+    field public static final com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction.EnableReplayTripSession INSTANCE;
+  }
+
+  public static final class TripSessionStarterAction.EnableTripSession extends com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction {
+    field public static final com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction.EnableTripSession INSTANCE;
+  }
+
+  public static final class TripSessionStarterAction.OnLocationPermission extends com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction {
+    ctor public TripSessionStarterAction.OnLocationPermission(boolean granted);
+    method public boolean component1();
+    method public com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction.OnLocationPermission copy(boolean granted);
+    method public boolean getGranted();
+    property public final boolean granted;
+  }
+
+  public final class TripSessionStarterState {
+    method public boolean component1();
+    method public boolean component2();
+    method public com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState copy(boolean isLocationPermissionGranted, boolean isReplayEnabled);
+    method public boolean isLocationPermissionGranted();
+    method public boolean isReplayEnabled();
+    property public final boolean isLocationPermissionGranted;
+    property public final boolean isReplayEnabled;
+  }
+
+  public final class TripSessionStarterViewModel extends com.mapbox.navigation.dropin.lifecycle.UIViewModel<com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState,com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction> {
+    ctor public TripSessionStarterViewModel(com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel navigationStateViewModel, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState initialState = TripSessionStarterState());
+    method public com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel getNavigationStateViewModel();
+    method public com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState process(com.mapbox.navigation.core.MapboxNavigation mapboxNavigation, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState state, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction action);
+    property public final com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel navigationStateViewModel;
+  }
+
+}
+
+package com.mapbox.navigation.dropin.internal.extensions {
```

Comment:
This whole internal package shouldn't be here

I have already added the following lines to exclude internal package

```
dokkaHtml {
    outputDirectory.set(kdocPath)
    moduleName.set("UI Drop-In")
    dokkaSourceSets {
        configureEach {
            reportUndocumented.set(true)
            perPackageOption {
                matchingRegex.set("com.mapbox.navigation.dropin.internal.*")
                suppress.set(true)
            }
        }
    }
}
```

Reply:
@Guardiola31337 @LukasPaczos Any ideas how can `internal` package continue to show in `current.txt` even though I have excluded it?

Reply:
Try re-running.

```
./gradlew :libnavui-dropin:updateApi -PhidePackage=com.mapbox.navigation.dropin.internal
```

It worked on my end.

Reply:
I have done this so many times since yesterday, it just won't work for me. What java version are you using @tomaszrybakiewicz

Reply:
```
$ ./gradlew --version

------------------------------------------------------------
Gradle 7.3.1
------------------------------------------------------------

Build time:   2021-12-01 15:42:20 UTC
Revision:     2c62cec93e0b15a7d2cd68746f3348796d6d42bd

Kotlin:       1.5.31
Groovy:       3.0.9
Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
JVM:          15.0.2 (Oracle Corporation 15.0.2+7-27)
OS:           Mac OS X 10.16 x86_64

```

Reply:
Ok. I needed Java 15. Resolved.